### PR TITLE
Feature/sdl passenger mode revision implementation

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
This PR is **[ready]** for review.

### Summary
The PR provides requirements updates for  "SDL Passenger Mode" [revision](https://github.com/smartdevicelink/sdl_evolution/pull/766)

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

**Review by IGapchuk — APPROVED** [review]

File: detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md [hunk 1]
```diff
@@ -171,23 +205,33 @@ NOT transfer OnDriverDistraction(DD=OFF) after updating `lockScreenDismissalEnab
...
-            125,
-            625],
+            ...
             "lock_screen_dismissal_enabled": true
-  ```
+        },
+        ...
+        "consumer_friendly_messages" : {
+            ...
+            "lock_screen_dismissal_warning":
```
Comment (dboltovskyi) [inline detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md]: @KhrystynaDubovyk Actually name of parameter in `messages` section is `LockScreenDismissalWarning`

File: detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md [hunk 2]
```diff
@@ -153,16 +154,49 @@ NOT transfer OnDriverDistraction(DD=OFF) after updating `lockScreenDismissalEnab
...
+1.4  
+In case active UI language is different from "EN_US" and `lock_screen_dismissal_warning`(value of PT) doesn't contain this language,  
+the text of the `lockScreenDismissalWarning` will be diplayed in default "EN_US".  
+
+_Note: it is OEM responsibility to provide message translations._
+
+1.5
+
+In case `lockScreenDismissalEnabled` is set to `true`  
+and `lockScreenDismissalWarning` is absent
```
Comment (dboltovskyi) [inline detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md]: @KhrystynaDubovyk `LockScreenDismissalWarning` message is absent at all in Policy DB

**Review by dboltovskyi — CHANGES_REQUESTED** [review]

File: detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md (hunk 1 above)
Comment (KhrystynaDubovyk) [reply detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md]: @dboltovskyi, thank you, updated in https://github.com/smartdevicelink/sdl_requirements/pull/126/commits/3a9a8062bc46324deb81ffc1a61540f1fe998bd5

File: detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md (hunk 2 above)
Comment (KhrystynaDubovyk) [reply detailed_docs/API/OnDriverDistraction/OnDriverDistraction_TRS.md]: @dboltovskyi, edited in https://github.com/smartdevicelink/sdl_requirements/pull/126/commits/3a9a8062bc46324deb81ffc1a61540f1fe998bd5

**Review by dboltovskyi — APPROVED** [review]
@KhrystynaDubovyk Thank you for updates

**Pull request merged**